marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/merging/merging-function-arguments.xqy

```diff
@@ -0,0 +1,168 @@
+      "maxValues": "1"
+    }
+  },
+  "tripleMerge": object-node {
+    "at": "/test/suites/data-hub/5/smart-mastering/merging/test-data/javascriptMergingFunctions.sjs",
+    "function": "quickStartMergeTriples"
+  }
+};
+
+declare variable $quickstart-xquery-options := object-node {
+  "propertyDefs": array-node {
+    object-node {
+      "property": object-node {
+        "name": "property1",
+        "path": "/property"
+      }
+    }
+  },
+  "algorithms": object-node {
+    "custom": array-node {
+      object-node {
+          "name": "quickStartMergeProperties",
+          "namespace": "http://marklogic.com/smart-mastering/xqueryMerging",
+          "at": "/test/suites/data-hub/5/smart-mastering/merging/test-data/xqueryMergingFunctions.xqy",
+          "function": "quickStartMergeProperties"
+        }
+    }
+  },
+  "merging": array-node {
+    object-node {
+      "propertyName":"property1",
+      "algorithmRef": "quickStartMergeProperties",
+      "maxValues": "1"
+    }
+  },
+  "tripleMerge": object-node {
+    "namespace": "http://marklogic.com/smart-mastering/xqueryMerging",
+    "at": "/test/suites/data-hub/5/smart-mastering/merging/test-data/xqueryMergingFunctions.xqy",
+    "function": "quickStartMergeTriples"
+  }
+};
+
+declare variable $hub-central-javascript-options := object-node {
+  "mergeRules": array-node {
+    object-node {
+      (: using documentXPath to avoid the need to load an entity  :)
+      "documentXPath":"/property",
+      "mergeModulePath": "/test/suites/data-hub/5/smart-mastering/merging/test-data/javascriptMergingFunctions.sjs",
+      "mergeModuleFunction": "hubCentralMergeProperties",
+      "maxValues": "1"
+    }
+  },
+  "tripleMerge": object-node {
+    "at": "/test/suites/data-hub/5/smart-mastering/merging/test-data/javascriptMergingFunctions.sjs",
+    "function": "hubCentralMergeTriples"
+  }
+};
+
+declare variable $hub-central-xquery-options := object-node {
+  "mergeRules": array-node {
+    object-node {
+      (: using documentXPath to avoid the need to load an entity  :)
+      "documentXPath":"/property",
+      "mergeModulePath": "/test/suites/data-hub/5/smart-mastering/merging/test-data/xqueryMergingFunctions.xqy",
+      "mergeModuleNamespace": "http://marklogic.com/smart-mastering/xqueryMerging",
+      "mergeModuleFunction": "hubCentralMergeProperties",
+      "maxValues": "1"
+    }
+  },
+  "tripleMerge": object-node {
+    "at": "/test/suites/data-hub/5/smart-mastering/merging/test-data/xqueryMergingFunctions.xqy",
+    "namespace": "http://marklogic.com/smart-mastering/xqueryMerging",
+    "function": "hubCentralMergeTriples"
+  }
+};
+
+declare variable $test-docs := (
+  document { object-node { "property": "val1" } },
+  document { object-node { "property": "val2" } }
+);
+
+(
+  (: Test XQuery triple functions :)
+  try {
+    merging-impl:build-final-triples(merging-impl:compile-merge-options($quickstart-xquery-options), $test-docs, ()),
```

Comment:
Assuming that the triples are drawn from test-docs, I think this would be a much more effective test if some assertions were made on the output of this function call. For example, looking at the function signature alone raises the question of - if the same triples are in multiple test docs, are duplicates discarded? Or is this just building a sequence of triples, regardless of what's in the triples? Assertions on the output would answer that.